src/lib/modules/blockchain_process/simulator.js

```diff
@@ -86,6 +86,10 @@ class Simulator {
 
     if(useProxy){
       let ipcObject = new Ipc({ipcRole: 'client'});
+      if (this.blockchainConfig.wsRPC) {
+        return new Proxy(this.proxyIpc).serve(host, port, true, this.blockchainConfig.wsOrigins, []);
```

Comment:
We could move this above line 88 since we don't use `ipcObject`

Reply:
updated, it was a typo

Reply:
Oh hahaha